src/pf-dropdown/pf-dropdown.spec.js

```diff
@@ -24,7 +24,34 @@ describe("Patternfly Dropdown Component Test", function () {
 
     customElement.addEventListener('hidden.bs.dropdown', function () {
       expect(button.parentNode.classList.contains('open')).toBe(false);
-      done();
     }, false);
   });
+
+  it("disabled button should not open dropdown", function () {
+    var button = customElement.querySelector('.btn');
+    button.classList.add('disabled');
+    customElement.toggle();
+    expect(button.parentNode.classList.contains('open')).toBe(false);
+    button.dispatchEvent(new MouseEvent('click'));
+    expect(button.parentNode.classList.contains('open')).toBe(false);
+  });
+
+  it("should select an element", function () {
+    var buttonText = customElement.querySelector('.btn');
+    var item = customElement.querySelector('ul.dropdown-menu > li:nth-child(2)');
+    expect(buttonText.innerText).toBe('Dropdown');
+    expect(item.innerText).toBe('Item 1');
+  });
+
+  it("should not click disabled item", function () {
```

Comment:
Try testing an enabled menu option, first. You should be able to fire a click event and test for changes (e.g., which option was selected). You can even attach an event listener and test if it was called.

Creating your own event is not a true test of the underlying functionality. That is, testing event.defaultPrevented only indicates that you called preventDefault() yourself. 

I'm thinking the anchor is not disabled properly? See below.